vectorial_model.py

```diff
@@ -0,0 +1,141 @@
+import math
+from collections import Counter
+from collections import OrderedDict
+
+from collection_processing import article_tokenize_other, lemmatize, remove_stop_words
+
+
+def remove_non_index_term(query, inverted_index):
+    """"Remove tokens that are not in index from query."""
+    query_filtered = []
+    for token in query:
+        if token in inverted_index:
+            query_filtered.append(token)
+    return query_filtered
+
+
+def pre_processed_query(query, inverted_index, stop_words):
+    """
+    Process raw query for vectorial vectorial query.
+
+    Tokenize, filter stop words and lemmatize query.
+    :param query: (string) query
+    :param inverted_index:
+    :return:
+    """
+    tokenized_query = article_tokenize_other(query)
```

Comment:
Why not using the simple tokenizer ?
Reminder: the collection is [already tokenized](https://docs.google.com/document/d/1Krhso_J-mNjL0KfbBBxnJAuUKtZ84MZpStHKXOVT0zA/edit?usp=sharing)

Reply:
In the lab the regex tokenizer is used, I don’t think it really makes much of a difference.

Reply:
It probably won't but I think it is better to tokenize everything the same way.